fastai/text/learner.py

```diff
@@ -65,7 +73,14 @@ def clean_raw_keys(wgts):
 
 # Cell
 #For previous versions compatibility, remove for release
-def load_model_text(file, model, opt, with_opt=None, device=None, strict=True):
+def load_model_text(
+    file:str, # File name of saved text model
+    model, # Model Architecture
+    opt:Optimizer, # `Optimizer` used to fit the model
+    with_opt:bool=None, # enable to load `Optimizer` state
```

Comment:
```suggestion
    with_opt:bool=None, # Enable to load `Optimizer` state
```

Reply:
On another note, is there a reason this is set to None rather than True?  This probably wouldn't be appropriate to change here, but I think this would be a nice chance to fastai unless I am missing the Intent of setting this to None

Reply:
I believe `load_model_text` exists only for backwards compatibility. I have never used it before as well. `load_model` in Learner has `with_opt` param set to True as you mentioned.